src/components/common/TopDrops/styles.js

```diff
@@ -0,0 +1,125 @@
+export default theme => ({
+  root: {
+    padding: [20, 0, 0],
+    fontFamily: theme.typography.secondary,
+    fontStyle: 'italic',
+    textAlign: 'center',
+    color: '#fff',
+    position: 'relative',
+    height: 150
+  },
+
+  //CONTENT AREA
+  content: {
+    overflow: 'hidden',
+    position: 'relative',
```

Comment:
This position probably is unnecessary.

Reply:
this element have childrens with position absolute to this element, not from the parent, when we remove it we produce visual errors